.travis.yml

```diff
@@ -11,10 +11,12 @@ install:
 - pip install -U tox
 matrix:
   include:
-  - python: 3.4.4
-    env: TOXENV=py34
   - python: 3.5
     env: TOXENV=py35
+  - python: 3.6
+    env: TOXENV=py36
+  - python: 3.7
```

Comment:
in order for travis to be able to download python 3.7 you should add `dist: xenial`
see https://github.com/travis-ci/travis-ci/issues/9815 for more info